src/Bswup/Bit.Bswup/Styles/bit-bswup.progress.css

```
@@ -54,3 +54,42 @@
+
+.bit-bswup-error-title {
+    font-size: 16px;
+    font-weight: 600;
+    margin: 0 0 6px 0;
+}
+
+.bit-bswup-error-message {
+    font-size: 13px;
+    margin: 0 0 8px 0;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)